### PR TITLE
Add `FindOptions.Backwards`

### PR DESCRIPTION
**Pull request by mialbu**
Solves #939

### TODOs
- [x] Add int-test for this find option

### CODE REVIEW

**Comment by mialbu** [comment]
@csmuller
The new `FindOptions.Backwards` is out of bounds for a `byte`, thus I declared it as `int` in Java. I'd leave the other `FindOptions` as `byte`s, do you agree, or would you change them to `int`s as well?

File: int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ public static io.neow3j.devpack.List<ByteString> findWithFindOptionValuesOnly()
 
         public static io.neow3j.devpack.Map<ByteString, ByteString> findWithFindOptionDeserializeValues() {
             Storage.put(ctx, hexToBytes("0102"), hexToBytes("102030"));
-            byte findOption = FindOptions.DeserializeValues & FindOptions.PickField0;
+            int findOption = FindOptions.DeserializeValues & FindOptions.PickField0;
```
Comment (csmuller) [inline int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java:1241]: Don't you think we should have at least one test that tests the new Backwards FindOption?
After all the discussions we had, I don't trust this stuff.

**Review by csmuller — CHANGES_REQUESTED** [review]

**Comment by csmuller** [comment]
> The new FindOptions.Backwards is out of bounds for a byte, thus I declared it as int in Java. I'd leave the other FindOptions as byte, do you agree, or would you change them to int as well?

Agree

File: int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java (hunk 1 above)
Comment (mialbu) [reply int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java:1241]: Yes, I thought about adding an integration test. Though, I’ll need to wait for neo3-privatenet-docker to update to Neo v3.6.0 including this new FindOption.

Adding a unit test doesn‘t add any value imo.

If you agree, please change this PR to a draft PR. Then, I‘ll await the Neo 3.6.0 release to continue on this issue.

Comment (csmuller) [reply int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java:1241]: ah ye right. I always forget how tedious our dependencies for a proper release are.

Comment (csmuller) [reply int-tests/src/test-integration/java/io/neow3j/compiler/StorageIntegrationTest.java:1241]: It seems that I can't change the PR to a draft. @mialbu can you do that?

**Comment by mialbu** [comment]
Waiting for Neo to release v3.6.0 and neo3-privatenet-docker to update. Then, I'll add an integration test to test the new FindOption.

**Review by csmuller — APPROVED** [review]
Nicolas Cage Hair - CiTLZWskt7Fu

**Pull request merged**